nebula/ui/components/qmldir

```diff
@@ -17,13 +17,8 @@ VPNChevron 0.1 VPNChevron.qml
 VPNClickableRow 0.1 VPNClickableRow.qml
 VPNCollapsibleCard 0.1 VPNCollapsibleCard.qml
 VPNConnectionInfo 0.1 VPNConnectionInfo.qml
```

Comment:
I still don't understand what goes into Nebula and what does not.
Most of these files should not be in nebula. What do you think?

Reply:
More complex and not reusable components should be probably not part of nebula. I think cleaning that up would better be part of the design system and theming efforts.